dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@ protected void postProcessAfterScopeModelChanged(ScopeModel oldScopeModel, Scope
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Optimization Issue**: Null check added for 'registryConfig' before accessing its method ensures null safety and avoids potential NullPointerException. <br> **Fix**: Include null checks before method invocations on objects that could potentially be null. <br> **Code Suggestion**: 
 ```
 -                if (registryConfig.getScopeModel() != applicationModel) {
 +                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
 ```